bundles/org.openhab.binding.boschindego/README.md

```diff
@@ -126,6 +126,13 @@ Number:Area              Indego_GardenSize         { channel="boschindego:indego
 Image                    Indego_GardenMap          { channel="boschindego:indego:singlekey:lawnmower:gardenMap" }
 ```
 
+---
+**NOTE**
+
+Currently some models (verified with Indego S+500 and Indego M+700) the channel `batteryVoltage` is not properly populated with the actual battery voltage but with the `batteryLevel`.
+
+---
+
```

Comment:
This is placed in the middle of the full example chapter. I would suggest to add it to the existing battery footnote in line 58.

Reply:
Agreed.